### PR TITLE
remove deploy dir due to sdk upgrade

### PR DESCRIPTION
**Pull request by songleo**
https://github.com/open-cluster-management/backlog/issues/10604
/hold

### CODE REVIEW

File: bundle/manifests/multicluster-observability-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -347,7 +347,7 @@ spec:
                   value: /usr/local/manifests
                 - name: SPOKE_NAMESPACE
                   value: open-cluster-management-addon-observability
-                image: quay.io/open-cluster-management/multicluster-observability-operator:2.3.0-SNAPSHOT-2021-03-16-13-59-25
+                image: quay.io/open-cluster-management/multicluster-observability-operator:latest
```
Comment (morvencao) [inline bundle/manifests/multicluster-observability-operator.clusterserviceversion.yaml:350]: Not sure if the `tag` here matters, others look good!

**Review by morvencao — COMMENTED** [review]
/lgtm

File: bundle/manifests/multicluster-observability-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (songleo) [reply bundle/manifests/multicluster-observability-operator.clusterserviceversion.yaml:350]: I think this tag is a placeholder.

**Pull request merged**